src/main/java/com/converter/AbstractConverter.java

```diff
@@ -0,0 +1,17 @@
+package com.converter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractConverter<S, T> implements IConverter<S, T> {
```

Comment:
All converters will transfer common for Entity and DTO field _id_ - try to implement this converting in abstract converter to not duplicate this logic in every converter